src/frontend/Player/createVideojsPlayer.ts

```diff
@@ -171,13 +177,37 @@ export const createVideojsPlayer = (
     xapiStatement.paused({
       time: player.currentTime(),
     });
+    if (hasAttendance) {
+      throttledAttendance();
+    }
   });
 
   /**************** Seeked statement ***********************/
+  const hasAttendance =
+    video.live_state &&
+    video.live_state === liveState.RUNNING &&
+    getDecodedJwt().permissions.can_update === false;
+  const throttledAttendance = throttle(() => {
+    const attendance = {
+      [Date.now()]: {
```

Comment:
Why are we using the timestamp as a key and also in the send object ?

Reply:
it's just by prevention because we don't really know how data will be handled in the back and didn't want to miss a chance to record data this way to save computing time